pkg/resource/job/list.go

```diff
@@ -128,6 +130,7 @@ func GetJobListFromChannels(channels *common.ResourceChannels, dsQuery *datasele
 	return jobList, nil
 }
 
+// ToJobList returns a list of all Jobs in the cluster reading required resource list once from the channel.
```

Comment:
what about the following description
// ToJobList returns a list of Jobs in the cluster by reading required resource list returned from the channel.